code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -514,16 +538,49 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/L, metabolis
 /datum/reagent/peridaxon/overdose_crit_process(mob/living/L, metabolism)
 	L.apply_damages(2*REM, 6*REM, 6*REM)
 
+
+/datum/reagent/medicine/peridaxon_plus
+	name = "Peridaxon"
+	description = "Used to repair internal organs. Significant toxic and stun side effects."
+	color = "#C845DC"
+	overdose_threshold = REAGENTS_OVERDOSE/30
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL/25
```

Comment:
Use multiplications instead of divisions whenever possible.

Reply:
fiine.
0.03333333333333333333333333333333333333333333333333333333333333333333333333 and 0.4 it is

Reply:
done